jinja/index.html

```diff
@@ -81,17 +81,17 @@ <h5>Demos <i class="fas fa-angle-double-right"></i></h5>
                 </a>
             </div>
             <div class="col-lg-4 mb-5">
-                <a href="https://qhack.ai">
-                    <div class="card card-image" style="background-image: url(img/qhack.png); background-size: cover;">
+                <a href="/carnival/index.html">
```

Comment:
This is taking me nowhere in the CircleCI build

Reply:
It still does not work for me. The other cards seem to be directly linking to the page, e.g. <a href="https://pennylane.ai/qml/demonstrations.html">, would this fix it?

Reply:
Thanks for catching. This is something that the server should do the redirect if it's been set up, but the CircleCI build doesn't add this quality of life improvement. 

Safest is to hardcode the full URL. The downside of that is it would actually take the reviewers away from the build page and to the "live" version, effectively hiding some of the changes from view unless they specifically entered the URLs

Reply:
fixed now